sdk-docs/reference/npm-libraries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: npm libraries
+category: 61fcd8e1a448f5004215317c
+---
+
+> 🚧 Beta
+> 
+> This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
+
+# npm Libraries
+
+The npm libraries that make up the SDK can be categorized into three types: vanilla JS libraries, tailwind-based styling libraries, and React libraries. 
+You can browse all of our packages on [npmjs.com](https://www.npmjs.com/search?q=%40trackunit) too by searching for @trackunit.
+
+
+## Vanilla JS Libraries
+While we recommend building Iris Apps with React, the following libraries enable you to use whatever framework you want (even plain Javascript).
+
+| Library | Usage |
+| --- | --- |
+| [@trackunit/iris-app](https://www.npmjs.com/package/@trackunit/iris-app) | Core library for building Iris applications with any JS framework. It's a [Nx](https://nx.dev/)  plugin that adds helpful generators used to setup your app.|
+| [@trackunit/iris-app-api](https://www.npmjs.com/package/@trackunit/iris-app-api) | A small package containing reusable types. TODO: what is the difference between this and runtime-core-api?|
+| [@trackunit/iris-app-runtime-core](https://www.npmjs.com/package/@trackunit/iris-app-runtime-core) | TODO |
+| [@trackunit/iris-app-runtime-core-api](https://www.npmjs.com/package/@trackunit/iris-app-runtime-core-api) | A small package containing reusable types. TODO: what is the difference between this and iris-app-api? |
```

Comment:
@trackunit/iris-app-api -> Contains the types for the main parts of  Iris APP SDK, including manifest and so on.

@trackunit/iris-app-runtime-core -> Is the Library containg the code for the runtimes that are exposed to iris apps. These facilitate communication with the manager and other things like seting up the rest api.
Some runtimes might be part of other packages, this is the CORE once, that will be in all iris apps. 

@trackunit/iris-app-runtime-core-api -> Types for the @trackunit/iris-app-runtime-core package.